packages/augur-ui/src/modules/common/labels.tsx

```diff
@@ -632,20 +633,31 @@ export const InReportingLabel = (props: InReportingLabelProps) => {
   );
 };
 
-export const PendingLabel = () => (
-  <span className={Styles.PendingLabel} data-tip data-for={"processing"}>
-    Processing <ClipLoader size={8} color="#ffffff" />
-    <ReactTooltip
-      id={"processing"}
-      className={TooltipStyles.Tooltip}
-      effect="solid"
-      place="top"
-      type="light"
-      data-event="mouseover"
-      data-event-off="blur scroll"
-    >
-      You will receive an alert when the transaction has finalized.
-    </ReactTooltip>
+interface PendingLabelProps {
+  status?: string;
+}
+
+export const PendingLabel = (props: PendingLabelProps) => (
+  <span className={classNames(Styles.PendingLabel, {[Styles.Failure]: props.status && props.status === TXEventName.Failure})} data-tip data-for={"processing"}>
+    {(!props.status || props.status === TXEventName.Pending) &&
+      <>
+        <span>Processing <ClipLoader size={8} color="#ffffff" /></span>
```

Comment:
can style be passed into ClipLoader rather than hardcoded color?

Reply:
the ClipLoader is a react package so dnt think

Reply:
ok, lets go with it.